.pre-commit-config.yaml

```diff
@@ -63,4 +111,4 @@ repos:
       additional_dependencies:
         # Make sure we use fmf compatible with tmt in the repo, not the 1.26.0 version.
         - "fmf>=1.3.0"
-        - pint==0.20
+        - "pint<0.20"
```

Comment:
Why are we lowering the `pint` version?

Reply:
Added a comment in f3ee8338. Pint >= 0.20 requires changes in tmt.hardware - better suited for another patch.

Reply:
Ack, thanks!